tools/get_release_contributors.py

```diff
@@ -5,8 +5,15 @@
 May need a Github Personal access token to avoid hitting rate limits.
 
 Usage:
-./tools/get_release_contributors.py --from release --to translate3_7_0 --github-token YOUR_TOKEN --repo /path/to/your/jamuluswebsite
-./tools/get_release_contributors.py --from r3_6_2 --to r3_7_0rc1 --github-token YOUR_TOKEN --repo /path/to/your/jamulus
+
+## Website repo (jamulussoftware/jamuluswebsite)
+- cd jamuluswebsite
+- git log release # Find commit ID of previous version. Search for RELEASE:
```

Comment:
Using `--graph` might make the log easier (not sure on that).  Also specifying a `--format` that limits the result to one line per commit, so long as the info is sufficient (if you're not generally a git user, you may not be familiar with configuring your got log format). `RELEASE:` should probably be in quotes to show it's a literal to search for.

Reply:
> Using `--graph` might make the log easier (not sure on that).

Hm, tried with `--graph`, not too convinced about the benefits in this case.

> Also specifying a `--format` that limits the result to one line per commit, so long as the info is sufficient (if you're not generally a git user, you may not be familiar with configuring your got log format).

Added `--oneline`

> `RELEASE:` should probably be in quotes to show it's a literal to search for.

Done.